'PBXProject' does not contain a definition for 'AddFileToEmbedFrameworks'

I'm using Unity 2021.3.27f1 and I got this error:
Editor/PurchaselyPostProcessor.cs(47,12): error CS1061: 'PBXProject' does not contain a definition for 'AddFileToEmbedFrameworks' and no accessible extension method 'AddFileToEmbedFrameworks' accepting a first argument of type 'PBXProject' could be found (are you missing a using directive or an assembly reference?)

I think that this function was moved into the Extensions sub package of PBXProject, I have fixed it by adding:

using UnityEditor.iOS.Xcode.Extensions;

--- a/Editor/PurchaselyPostProcessor.cs
+++ b/Editor/PurchaselyPostProcessor.cs
@@ -7,6 +7,7 @@ using UnityEditor.Callbacks;
 using UnityEngine;
 #if UNITY_IOS
 using UnityEditor.iOS.Xcode;
+using UnityEditor.iOS.Xcode.Extensions;
 #endif
 
 namespace Purchasely.Editor

